Dockerfile-centos

```
@@ -1,6 +1,7 @@
 FROM centos
 
-RUN yum update -y && yum install -y epel-release && yum install -y git curl dpkg java java-devel unzip && yum clean all
+RUN yum update -y && yum install -y epel-release && yum install -y git curl dpkg java java-devel unzip which && yum clean all
```

I do not mind keeping `which` here. But should it be actually excluded from the PR?

```suggestion
RUN yum update -y && yum install -y epel-release && yum install -y git curl dpkg java java-devel unzip && yum clean all
```

---

All docker images with Jenkins have `which`. Without it the behavior of "gradle" or "maven" does not work properly, they can do it with `JAVA_HOME`, but which one seems cleaner.

---

agreed, let's keep it as a separate change